include/deal.II/hp/dof_handler.h

```diff
@@ -182,7 +182,7 @@ namespace hp
      *
      * @ingroup Iterators
      */
-    typedef typename ActiveSelector::cell_iterator        cell_iterator;
+    typedef TriaIterator < dealii::DoFCellAccessor < DoFHandler < dim, spacedim >, false > >        cell_iterator;
```

Comment:
I tried this as well -  VS gives an error C2882 (illegal use of namespace identifier in expression).